AlphaWallet/Wallet/ViewControllers/ImportWalletViewController.swift

```diff
@@ -128,13 +130,15 @@ class ImportWalletViewController: UIViewController, CanScanQRCode {
         watchAddressTextField.delegate = self
         watchAddressTextField.returnKeyType = .done
 
+        let col1 = [mnemonicTextView.statusLabel, mnemonicCountLabel].asStackView()
```

Comment:
`row2` instead of `col1` maybe? So we have:

```
[
  mnemonicTextView.label,
  .spacer(height: 4),
  mnemonicTextView,
  .spacer(height: 4),
  row2,
]
```

Reply:
yep you right.